src/fsharp/service/ServiceParsedInputOps.fsi

```diff
@@ -44,6 +45,11 @@ type public CompletionContext =
     /// Completing pattern type (e.g. foo (x: |))
     | PatternType
 
+    /// Completing union case fields declaration (e.g. 'A of stri|' but not 'B of tex|: string')
+    | UnionCaseFieldsDeclaration
+
+    | TypeAbbreviation
```

Comment:
I think add a comment that this covers both type abbreviation `type X = int$` and single-case-unions-without-the-bar `type X = AB$` , and perhaps rename to cover that.